docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md

```diff
@@ -0,0 +1,61 @@
+# Control Definitions
+
+A control definitions document is a detailed specification of security measures designed to mitigate risks and ensure compliance with established standards and frameworks. It must contain a list of all controls that pertain to a specific service category along with relevant supporting information.
+
+A control definition should be created for each service in the CCC Taxonomy, with each control in the document mapped to a feature on that service. A set of behavioral test definitions may be provided alongside the control definition in Gherkin to support validation of the controls.
+
+## Control Definition Format
+
+In order to create a cohesive standard that is readily useful to end users, controls must be indistinguishable from eachother in format, style, and tone. As such, all controls must match the layout presented in the [control template](../templates/control.md) prior to release.
+
+This following list outlines the values necessary to create a new control definition using the control template.
+
+- `Control Description` - This is a 1 to 2 sentence description of this control, and what it achieves.
+- `Control Incrementer` - A sequential number or identifier for each control. This must not be changed or reused once assigned.
+- `Control Title` - This is a 3 to 10 word title that succintly describes the control.
+- `Corresponding Control ID` - The identifiers for other controls that are related or linked to the current control.
+- `NIST CSF Category` - This indicates the relevant category from the NIST Cybersecurity Framework that the control addresses.
+- `NIST CSF ID` - The specific ID from the NIST Cybersecurity Framework that corresponds to the control.
+- `Service Name` - This is the name of the service, which must match the name value provided by the corresponding taxonomy entry.
+- `Service Name Shorthand` - This is the short version of the service name, such as "OS" for "Object Storage", which should match the corresponding taxonomy entry.
+- `Service Identifier` - This is the unique identifier assigned in the taxonomy entry for a service.
+- `Test Requirement Description` - A detailed description of the testing requirements needed to validate the control’s implementation.
```

Comment:
In the object storage catalog, _Test Requirement Description_ is down as _Testing Requirements_ which is made up of a list of validations that need to be performed. Each testing requirement item links to a gherkin feature in the control's corresponding gherkin feature file: https://github.com/finos/common-cloud-controls/blob/main/services/storage/object/controls.md?plain=1#L29  

I think this works quite well. If we agree, I think we'd need that detail here.